### DISCUSSION TITLE
Cosmological models are built on a simple, century-old idea, but new observations demand a radical rethink

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#87] [Cosmology Models]
[https://phys.org/news/2023-06-cosmological-built-simple-century-old-idea.html](https://phys.org/news/2023-06-cosmological-built-simple-century-old-idea.html)

"As new telescopes, both on Earth and in space, deliver ever more precise images, and astronomers discover massive objects such as the [giant arc of quasars](https://www.bbc.com/future/article/20230302-the-giant-arcs-that-may-dwarf-everything-in-the-cosmos), this foundation is increasingly challenged."

**Comment by OlliSant** [comment]
As a web of galaxies the universe is homogeneous, not in smaller unites. The universe is the universe, the whole universe, not any smaller part of the universe. This is always forgotten in the theories. The universe is much bigger than the observable universe.
